engine/gamesys/src/gamesys/components/comp_tilegrid.cpp

```diff
@@ -805,8 +805,8 @@ namespace dmGameSystem
             }
         }
 
-        if (render_list != write_ptr)
-            dmRender::RenderListSubmit(render_context, render_list, write_ptr);
```

Comment:
The `RenderListAlloc()` must nowadays be matched with a call to `RenderListSubmit()`